Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -93,20 +100,77 @@ namespace Terrain
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxRenderRadius,
+                        "Detail clipmap max render radius: m",
+                        "Max render radius that the lowest resolution clipmap can cover. \n"
+                        "For example, 1000 means the clipmaps render 1000 meters at most from the view position.")
+                        ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 100.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 2000.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 10000.0f)
                     ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMaxResolution,
-                        "Macro clipmap max resolution (in texels per meter)",
+                        "Macro clipmap max resolution: texels/m",
                         "The resolution of the highest resolution clipmap in the stack.")
-                        ->Attribute(AZ::Edit::Attributes::Min, 0.5f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
                         ->Attribute(AZ::Edit::Attributes::SoftMin, 2.0f)
-                        ->Attribute(AZ::Edit::Attributes::Max, 16.0f)
-                        ->Attribute(AZ::Edit::Attributes::SoftMax, 128.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 10.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 100.0f)
                     ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_detailClipmapMaxResolution,
-                        "Detail clipmap max resolution (in texels per meter)",
+                        "Detail clipmap max resolution: texels/m",
                         "The resolution of the highest resolution clipmap in the stack.")
-                        ->Attribute(AZ::Edit::Attributes::Min, 0.5f)
-                        ->Attribute(AZ::Edit::Attributes::SoftMin, 256.0f)
-                        ->Attribute(AZ::Edit::Attributes::Max, 2048.0f)
-                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4096.0f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 10.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 512.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 2048.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 4096.0f)
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapScaleBase,
+                        "Macro clipmap scale base",
+                        "The scale base between two adjacent clipmap layers. \n"
+                        "For example, 3 means the (n+1)th clipmap covers 3^2 = 9 times to what is covered by the nth clipmap.")
```

Comment:
Suggestion: "For example, 3 means the (n+1)th clipmap covers 3^2 = 9 times the area covered by the nth clipmap."